src/main/resources/db/changelog/data/financialTransaction.csv

```diff
@@ -0,0 +1,9 @@
+id,wallet_id,transaction_type,amount,transaction_date,description
+1,1,INCOME,15000,2022-02-19T09:47:52.595721658,""
```

Comment:
What is the start/end character for data content for a single field in CSV for Liquibase data?
Is it ", ', | or else?
You know, if in the Liquibase config(?) we got another character this `""` gonna be paste just as String: `""` - not EMPTY_STRING value.